.changeset/thin-icons-scream.md

```diff
@@ -0,0 +1,26 @@
+---
+"@pnpm/resolve-dependencies": minor
+"@pnpm/merge-lockfile-changes": minor
+"@pnpm/package-requester": minor
+"@pnpm/directory-fetcher": minor
+"@pnpm/tarball-fetcher": minor
+"@pnpm/exec.pkg-requires-build": minor
+"@pnpm/hooks.read-package-hook": minor
+"@pnpm/lockfile-types": minor
+"@pnpm/prune-lockfile": minor
+"@pnpm/create-cafs-store": minor
+"@pnpm/lockfile-file": minor
+"@pnpm/fetcher-base": minor
+"@pnpm/headless": minor
+"@pnpm/deps.graph-builder": minor
+"@pnpm/node.fetcher": minor
+"@pnpm/core": minor
+"@pnpm/cafs-types": minor
+"@pnpm/types": minor
+"@pnpm/config": minor
+"@pnpm/store.cafs": minor
+"@pnpm/worker": minor
+"pnpm": minor
+---
+
+Add a field named `ignoredOptionalDependencies`. This is an array of strings. If an optional dependency has its name included in this array, it will be skipped, unless it is non-optional for some other packages.
```

Comment:
```suggestion
Add a field named `ignoredOptionalDependencies`. This is an array of strings. If an optional dependency has its name included in this array, it will be skipped.
```